lib/remote/filterutility.cpp

```diff
@@ -269,7 +269,7 @@ std::vector<Value> FilterUtility::GetFilterTargets(const QueryDescription& qd, c
 
 		if (query->Contains("filter")) {
 			String filter = HttpUtility::GetLastParameter(query, "filter");
-			std::unique_ptr<Expression> ufilter = ConfigCompiler::CompileText("<API query>", filter);
+			Expression::Ptr ufilter = static_cast<Expression::Ptr>(ConfigCompiler::CompileText("<API query>", filter).get());
```

Comment:
# Riddle

Why will this crash?

Reply:
I think, I don't need to `static_cast<>` the pointer.

Reply:
Would still crash.

Reply:
Replace `*.get()` by `*.release()`.